docs/guides/chip_tool_guide.md

```diff
@@ -0,0 +1,808 @@
+To list all `pairing` sub-commands, run the following command:
+
+```
+$ chip-tool pairing
+```
+
+### Interacting with Data Model clusters
+
+As mentioned in the [Using the CHIP Tool for Matter device testing](#using)
+section, executing the `chip-tool` command with a particular cluster name lists
+all operations supported for this cluster, as in the following command pattern:
+
+```
+$ chip-tool <cluster_name>
+```
+
+**Example of command:**
+
+```
+$ chip-tool binding
+```
+
+**Example of output:**
+
+```bash
+[1647502596.396184][411686:411686] CHIP:TOO: Missing command name
+Usage:
+  ./chip-tool binding command_name [param1 param2 ...]
+
+  +-------------------------------------------------------------------------------------+
+  | Commands:                                                                           |
+  +-------------------------------------------------------------------------------------+
+  | * command-by-id                                                                     |
+  | * read-by-id                                                                        |
+  | * read                                                                              |
+  | * write-by-id                                                                       |
+  | * write                                                                             |
+  | * subscribe-by-id                                                                   |
+  | * subscribe                                                                         |
+  | * read-event-by-id                                                                  |
+  | * subscribe-event-by-id                                                             |
+  +-------------------------------------------------------------------------------------+
+[1647502596.396299][411686:411686] CHIP:TOO: Run command failure: ../../examples/chip-tool/commands/common/Commands.cpp:84: Error 0x0000002F
+```
+
+According to this list, the `binding` cluster supports operations such as read
+or write. Attributes from that cluster can also be subscribed by the controller,
+which means that the CHIP Tool will receive notifications, for instance when the
+attribute value is changed or a specific event happens.
+
+#### Examples
+
+This section lists examples of the CHIP Tool commands dedicated to specific use
+cases.
+
+##### Writing ACL to the `accesscontrol` cluster
+
+The Access Control List (ACL) concept allows to govern all Data Model
+interactions (such as read attribute, write attribute, invoke command). For more
+information about ACL, see
+[Access Control Guide](https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/access-control-guide.md).
+
+To write ACL to the `accesscontrol` cluster, use the following command pattern:
+
+```
+$ chip-tool accesscontrol write acl <acl_data> <node_id> <endpoint_id>
+```
+
+In this command:
+
+-   _<acl_data\>_ is the ACL data formatted as a JSON array.
```

Comment:
Since it's a rather complex structure, some example would be pretty useful.

Reply:
The assumption was that the concrete commands shall be a part of guides for exemplary applications. But I have no strong opinion here, @greg-fer what do you think?

Reply:
I think an example is most welcome.